### DISCUSSION TITLE
[Packagekit] - www.cve.org replacement by www.cvedetails.com ?

### DISCUSSION THREAD

**Discussion by lau453** [cockpit-project/cockpit#20241] [Ideas]
Hi,

Refering to the latest pull request here https://github.com/cockpit-project/cockpit/pull/19788, I would suggest to even replace www.cve.org by www.cvedetails.com ?

Here is an example of URL for a given CVE : https://www.cvedetails.com/cve/CVE-2019-9641/

I find out a lot more information there, at a glance

**Comment by jelly** [comment]
I prefer we keep linking to the official CVE website instead of a for me unknown CVE website from a Company I don't know and have no idea when they might change their website or market their own products on the CVE page.

Offtopic, I much prefer the Luxembourg CVE CERT website https://cve.circl.lu/cve/CVE-2019-9641

**Comment by adulau** [comment]
Maybe you want to use our new vulnerability-lookup which is a replacement for cve.circl.lu is [https://vulnerability.circl.lu/](https://vulnerability.circl.lu/) and the API endpoint is available https://vulnerability.circl.lu/doc
